assets/stylesheets/_components.scss

```diff
@@ -11,7 +11,7 @@
 @import 'auth/style';
 @import 'components/accordion/style';
 @import 'components/app-promo/style';
-@import 'components/author-selector/style';
+@import 'blocks/author-selector/style';
```

Comment:
I'm not re-alphabetizing yet explicitly in fear of regressions with blocks coming before ui components, and because I want to see what the coverage looks like for blocks currently scattered between components and section folders.